pkgs/by-name/bi/bitwarden-desktop/package.nix

```
@@ -93,6 +94,11 @@ in buildNpmPackage rec {
     libsecret
   ];
 
+  bitwardenDesktopNativeArch = {
+    aarch64 = "arm64";
+    x86_64  = "x64";
+  }.${stdenv.hostPlatform.parsed.cpu.name};
```

```suggestion
  bitwardenDesktopNativeArch = {
    aarch64 = "arm64";
    x86_64  = "x64";
  }.${stdenv.hostPlatform.parsed.cpu.name} or throw "bitwarden-desktop: unsupported CPU name ${stdenv.hostPlatform.parsed.cpu.name}";;
```

Make sure we give proper error messages for everyone else.